test/rulesets.jl

```diff
@@ -54,6 +54,18 @@ end
     @test simplify(Term(zero, [x + 2])) == 0
 end
 
+@testset "LiteralReal" begin
+    @syms x1::LiteralReal x2::LiteralReal
+    s = cos(x1 * 3.2) - x2 * 5.8 + x2 * 1.2
+    @eqtest s == cos(x1 * 3.2) - x2 * 5.8 + x2 * 1.2
+
+    # Prevents automatic simplification:
+    @eqtest s != cos(x1 * 3.2) - x2 * 4.6
+
+    # However, manual simplification should still work:
+    @eqtest simplify(s) == cos(3.2(x1^1)) - 4.6x2
```

Comment:
shouldn't this become

```
cos(3.2x1) - 4.6x2
```
?

My update does this....

Reply:
Ok I see it was leaving the `^1` before because of a missed case. I can safely change the test here I think.

Reply:
Although, it also changes it further. `LiteralReal` represents `cos(3.2(x1^1)) - 4.6x2` as `cos(3.2(x1^1)) - (4.6*x2)`. simplification turns it into `cos(3.2(x1^1)) + -4.6 * x2`